tests/integration/test_integration.py

```diff
@@ -121,7 +119,8 @@ def test_firehose_kinesis_to_s3(self):
         testutil.assert_objects(json.loads(to_str(test_data)), all_objects)
 
     def test_kinesis_lambda_sns_ddb_sqs_streams(self):
-        ddb_lease_table_suffix = '-kclapp'
+        ddb_lease_table_suffix = '-kclapp1'
```

Comment:
Not sure about this change. Note that `ddb_lease_table_suffix` is passed in to `kinesis_connector.listen_to_kinesis(...)` on line 144, which internally uses the KCL (Kinesis Client Library), which itself creates and uses a DynamoDB table to keep track of the offsets as the client is consuming Kinesis events over time.

In other words - the test `test_kinesis_lambda_sns_ddb_sqs_streams(..)` actually creates two DynamoDB tables - one for the test itself, and one table for keeping track of the Kinesis offsets. With the changes in this PR, only a single table is created and used for both. Is that on purpose? I think we should keep these tables separate, hence use two different names.

Reply:
(same for `test_lambda_streams_batch_and_transactions(..)` below)

Reply:
Sorry, the KCL calling in these tests are a bit dense. I'm assuming the table created is the stream name + ddb_lease, correct? 

My change was to get a second set of events from the dynamodb stream, but not necessary a fresh table ( although that would have been preferred). 

Are you suggesting that the test table name/stream name be changed instead of the lease suffix?

Reply:
I think my most recent commit does what you've asked.